doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -4,43 +4,34 @@ High Level Architecture
 Introduction
 ------------
 
-The Anuket Kubernetes Reference Architecture (RA) is intended to be an industry
-standard-independent Kubernetes reference architecture that is not tied to any
-specific offering or distribution. No vendor-specific enhancements are required
-to achieve conformance with the Anuket specifications.
-Conformance to these specifications can be ensured by doing the following:
-
-* Successfully running RC2.
-* Using upstream components or features that are developed by the open source
-  community.
-
-Using the RA2 Kubernetes-based architecture specifications, operators can deploy
-infrastructure that will run any  VNF or CNF that has successfully run on an
-RA2-conformant infrastructure. The purpose of this chapter is to outline all the
-components required to provide Kubernetes in a consistent and reliable way. The
-specification of how to use these components is detailed in Chapter 04
-:ref:`chapters/chapter04:component level architecture`.
-
-Kubernetes is already a well-documented and widely deployed open-source project
-managed by the Cloud Native Computing Foundation (CNCF). Full documentation of
-the Kubernetes code and project can be found at
-`https://kubernetes.io/docs/home/ <https://kubernetes.io/docs/home/>`__.
-The following chapters only describe the specific features required by the
-Anuket Reference Architecture, and how they are expected to be implemented.
-For information related to standard Kubernetes features and capabilities, see
-the standard Kubernetes documentation.
-
-While this reference architecture provides options for pluggable components,
-such as service mesh, the focus of the reference architecture
-is on the abstracted interfaces and features that are required for Telco-type
-workload management and execution.
-
-Chapter 5 of the Reference Model (RM) describes the hardware and software profiles
-that are descriptions of the capabilities and features that the Cloud Infrastructure
-provides to the workloads. Figure 5-3 in the RM (also shown below)
-depicts a high-level view of the software profile features that apply to each
-instance profile (basic and high-performance). For more information on the instance profiles,
-see :ref:`ref_model:chapters/chapter04:profiles`.
+The Anuket Kubernetes Reference Architecture (RA) is intended to be an industry standard-independent Kubernetes
```

Comment:
Let's use "Reference Architecture (RA2) for Kubernetes based cloud infrastructure" instead of "Kubernetes Reference Architecture (RA)"